packages/explorer/README.md

```diff
@@ -95,3 +95,80 @@ Install the `Live Server` extension on VSCode, then go to `localDevelopmentExamp
 `cd` into `packages/explorer` and run `npm run build:cjs-esm` to build cjs & esm files where ApolloExplorer & ApolloExplorer React are named exports.
 
 We have a React example app that uses our ApolloExplorer React component to render the embedded Explorer located in src/examples/react-example. To run this example, `npm run build` and `npm run start` in `react-example`. Make sure you delete the .parcel-cache folder before you rebuild for new changes. (TODO remove parcel caching)
+
+### Sequence Diagrams
+
+#### Connecting to unregistered graphs by directly passing in schema
+```mermaid
+sequenceDiagram
+    participant Parent as Parent Page
+    participant Embed as Embedded Explorer
+
+    note over Parent: Render iframe loading embed and start listening for messages.
+    Parent->>Embed: Embed loads with initial state <br>(gql operation, variables, headers, persistence preference etc) <br>in the query params of the iframe.
+    note over Embed: Loaded and sets initial state<br> from query params.
+    note over Embed: Start listening for messages
+
+    Embed->>Parent: Handshake message asking for schema
+    Parent->>Embed: Responds with schema
+
+    note over Embed: Loads in given schema to explorer
+
+    Embed --> Parent: Set up finished
+    
+    note over Embed: User submits operation
+    Embed->>Parent: Send request contents
+    note over Parent: Makes actual network via `handleRequest()`
+    Parent->>Embed: Send network response back
+    note over Embed: Processes and renders response
+```
+
+
+#### Connecting to registered graphs by authentication
+
+```mermaid
+sequenceDiagram
+    participant Parent as Parent Page
+    participant Embed as Embedded Explorer
+    participant Login as Studio Login Page at <br>studio.apollographql.com
+
+    note over Parent: Render iframe loading embed and start listening for messages.
+    Parent->>Embed: Embed loads with initial state <br>(gql operation, variables, headers, persistence preference etc) <br>in the query params of the iframe.
+    note over Embed: Loaded and sets initial state<br> from query params.
+    note over Embed: Start listening for messages
+
+    Embed->>Parent: Handshake message asking for graphRef, <br>account id & account invite token if provided
+        
+    Parent->>Embed: Responds with graphRef, account id, account invite token
+
+    Embed->>Parent: Authentication message asking for second half of<br> authentication token from parent page local storage
+
+    alt parent page has authentication token or graph is public
```

Comment:
thoughts on adding "when" or "if" to make the resulting chart a bit easier to read? 
```suggestion
    alt when parent page has authentication token or graph is public
```